pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -146,18 +150,26 @@ public class NamespaceService implements AutoCloseable {
 
     private final RedirectManager redirectManager;
 
-
+    @PulsarDeprecatedMetric(newMetricName = "pulsar.broker.lookup.redirect")
```

Comment:
Nice idea to put new metric here! We could use this to automatically create a conversion table